### PR TITLE
Docmosis aks DNS record

### PR DESCRIPTION
**Pull request by garethheath**
**Before creating a pull request make sure that:**

- [ x ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

### Change description ###

Create a DNS record for the docmosis deployment running in the shared services AKS cluster.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: environments/sandbox/ss-service-core-compute-sandbox-internal.yml [hunk 1]
```diff
@@ -28,4 +28,8 @@ A:
     record:
     - 10.140.15.250
     ttl: 300
+  - name: docmosis-aks
```
Comment (timja) [inline environments/sandbox/ss-service-core-compute-sandbox-internal.yml:31]: do you need to suffix it with -aks? it appears to be in a different zone to docmosis normally?

Comment (garethheath) [reply environments/sandbox/ss-service-core-compute-sandbox-internal.yml:31]: The suffix isn't required, I just wanted to make a clear distinction between the Docmosis services running on the virtual machines and those running in AKS.  Happy to change it.

Comment (garethheath) [reply environments/sandbox/ss-service-core-compute-sandbox-internal.yml:31]: ....This was only for testing

**Review by timja — APPROVED** [review]
fine for sandbox, I wouldn't use it with this name further probably. people tend to not bother removing the suffix later.

**Pull request merged**